walk-in-interview/src/main/webapp/common-functions.js

```diff
@@ -33,10 +33,9 @@ function getRequirementsList() {
  * @param {String} errorMessageElementId the element id for html error message div.
  * @param {String} msg the message that the error div should display.
  * @param {boolean} includesDefault whether the includes default message. Default to be true.
- * message should be included.
  */
 function setErrorMessage(errorMessageElementId, msg, includesDefault=true) {
```

Comment:
isn't the elementId always 'error-message'? What's the need for another param

Reply:
Element Id is dependent on specific html element, so maybe not good to simply regard as constant, same for `getJobDetailsFromUserInput`